client/packages/system/src/Stock/Components/InventoryAdjustment/InventoryAdjustmentForm.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { FC } from 'react';
+import {
+  TextWithLabelRow,
+  InputWithLabelRow,
+  InputWithLabelRowProps,
+  useTranslation,
+  Box,
+  NumericTextInput,
+  DialogButton,
+  useNotification,
+  AdjustmentDirectionInput,
+} from '@openmsupply-client/common';
+import { StockLineRowFragment, useInventoryAdjustment } from '../../api';
+import {
+  Adjustment,
+  InventoryAdjustmentReasonSearchInput,
+  usePackVariant,
+} from '../../..';
+import { InventoryAdjustmentDirectionInput } from './InventoryAdjustmentDirectionSearchInput';
+
+// TODO... use direction input in stocktake?
+const tempAsAdjustment = (direction: AdjustmentDirectionInput | null) => {
+  switch (direction) {
+    case AdjustmentDirectionInput.Addition:
+      return Adjustment.Addition;
```

Comment:
we already had the `Adjustment` enum from Stocktake, but only exists in the frontend. Felt like it will be simpler logic to do on both front and backend with enum that we send in the mutation (backend business logic can use this direction to determine if the reason is correct, rather than deriving direction from the delta between current and new num of packs?)

Curious for thoughts on this!